src/query/contact_manifolds/contact_manifolds_workspace.rs

```diff
@@ -65,7 +65,7 @@ impl DeserializableWorkspaceData {
             DeserializableWorkspaceData::CompositeShapeShapeContactManifoldsWorkspace(w) => {
                 Some(ContactManifoldsWorkspace(Box::new(w)))
             }
-            DeserializableWorkspaceData::Custom(_) => None,
+            DeserializableWorkspaceData::Custom => None,
```

Comment:
Do I understand that custom shapes are (currently) not supported by serialization/deserialization ? (at least here specifically)

Reply:
That is correct. And this is the same behavior as before.